openshift/app.dc.yaml

```diff
@@ -216,36 +216,6 @@ objects:
                     secretKeyRef:
                       key: password
                       name: getok-sc-keycloak-prod-secret
-                - name: SC_WEBADE_INT_USERNAME
-                  valueFrom:
-                    secretKeyRef:
-                      key: username
-                      name: getok-sc-webade-int-secret
-                - name: SC_WEBADE_INT_PASSWORD
-                  valueFrom:
-                    secretKeyRef:
-                      key: password
-                      name: getok-sc-webade-int-secret
-                - name: SC_WEBADE_TEST_USERNAME
-                  valueFrom:
-                    secretKeyRef:
-                      key: username
-                      name: getok-sc-webade-test-secret
-                - name: SC_WEBADE_TEST_PASSWORD
-                  valueFrom:
-                    secretKeyRef:
-                      key: password
-                      name: getok-sc-webade-test-secret
-                - name: SC_WEBADE_PROD_USERNAME
-                  valueFrom:
-                    secretKeyRef:
-                      key: username
-                      name: getok-sc-webade-prod-secret
-                - name: SC_WEBADE_PROD_PASSWORD
-                  valueFrom:
-                    secretKeyRef:
-                      key: password
-                      name: getok-sc-webade-prod-secret
```

Comment:
i'm guessing this will remove the webade secrets in openshift that we will no longer need

Reply:
Will remove them from the deployment but they stay in the secrets in the namespace(s).
Will clean up appropriate configs that aren't needed any more, but the **passwords for the webade clients** will need to remain, as those are existing clients for this team in the webade system that could be referenced again if needed in the future for any reason.

Reply:
ok. got it. thanks